cowbird/permissions_synchronizer.py

```diff
@@ -74,59 +74,178 @@ def __init__(self,
         available_services = ServiceFactory().services_cfg.keys()
         # Make sure that only active services are used
         self.services = {svc: svc_cfg for svc, svc_cfg in services.items() if svc in available_services}
-        self.mapping = [{svc: perms for svc, perms in mapping_pt.items() if svc in available_services}
-                        for mapping_pt in mapping]
+        self.resource_keys = {res_key: res for svc in self.services.values() for res_key, res in svc.items()}
```

Comment:
I think the name is accurate here. The resource keys are found in each service in the config.
Then it gets mappings that use those resource keys. So here, the service have resource keys, and the mappings also have resource keys. Resource keys represent values like `Geoserver_workspace` and `Thredds_workspace` found in the config example.

Reply:
I guess I should just rename `self.resource_keys` to `self.resources` then. It does make more sense since the dict actually contains resources, which are accessible by their respective resource_key.